plugins/storage/volume/solidfire/src/org/apache/cloudstack/storage/datastore/provider/SolidFireHostListener.java

```diff
@@ -91,18 +280,5 @@ public boolean hostConnect(long hostId, long storagePoolId) {
         assert (answer instanceof ModifyStoragePoolAnswer) : "ModifyStoragePoolAnswer expected ; Pool = " + storagePool.getId() + " Host = " + hostId;
```

Comment:
Off topic but I wanted to know what is the take on asserts? Personally I love asserts but not sure what the community thinks of it.